### ISSUE TITLE
[CitationStyleLanguage] Fatal error when previewing submission not assigned to any issue

### ISSUE THREAD

**Issue by jonasraoni** [pkp/pkp-lib#8755]
**Describe the bug**
```php
PHP Fatal error:  Uncaught TypeError: APP\issue\Repository::get(): Argument #1 ($id) must be of type int, null given, called in /var/www/html/plugins/generic/citationStyleLanguage/CitationStyleLanguagePlugin.php on line 430 and defined in /var/www/html/classes/issue/Repository.php:60
Stack trace:
#0 /var/www/html/plugins/generic/citationStyleLanguage/CitationStyleLanguagePlugin.php(430): APP\issue\Repository->get(NULL)
#1 /var/www/html/plugins/generic/citationStyleLanguage/CitationStyleLanguagePlugin.php(330): APP\plugins\generic\citationStyleLanguage\CitationStyleLanguagePlugin->getCitation(Object(APP\core\Request), Object(APP\submission\Submission), 'apa', NULL, Object(APP\publication\Publication))
#2 [internal function]: APP\plugins\generic\citationStyleLanguage\CitationStyleLanguagePlugin->getTemplateData('ArticleHandler:...', Array)
#3 /var/www/html/lib/pkp/classes/plugins/Hook.php(139): call_user_func_array(Array, Array)
#4 /var/www/html/lib/pkp/classes/plugins/Hook.php(112): PKP\plugins\Hook::run('ArticleHandler:...', Array)
#5 /var/www/html/pages/article/ArticleHandler.php(355): PKP\plugins\Hook::call('ArticleHandler:...', Array)
#6 [internal function]: APP\pages\article\ArticleHandler->view(Array, Object(APP\core\Request))
#7 /var/www/html/lib/pkp/classes/core/PKPRouter.php(465): call_user_func(Array, Array, Object(APP\core\Request))
#8 /var/www/html/lib/pkp/classes/core/PKPPageRouter.php(308): PKP\core\PKPRouter->_authorizeInitializeAndCallRequest(Array, Object(APP\core\Request), Array, false)
#9 /var/www/html/lib/pkp/classes/core/Dispatcher.php(163): PKP\core\PKPPageRouter->route(Object(APP\core\Request))
#10 /var/www/html/lib/pkp/classes/core/PKPApplication.php(372): PKP\core\Dispatcher->dispatch(Object(APP\core\Request))
#11 /var/www/html/index.php(21): PKP\core\PKPApplication->execute()
#12 {main}
  thrown in /var/www/html/classes/issue/Repository.php on line 60
```

**To Reproduce**
1. Enable the plugin
2. Create a submission, do not assign it to an issue
3. Preview the submission

**What application are you using?**
OJS 3.4.0rc1

**PRs**
- https://github.com/pkp/citationStyleLanguage/pull/105

**Comment by jonasraoni** [comment]
@asmecher PR
- https://github.com/pkp/citationStyleLanguage/pull/105

**Comment by asmecher** [comment]
Merged, thanks! We're getting very close to 3.4.0rc2, so please try to avoid extra cleanups/refactors (unless you're very sure you've fully tested them). Even adding formal typehints can easily cause unexpected fatal errors in boundary conditions.
